modules/test/playwright/helpers/HeadlessCommerceDeliveryCartApiHelper.ts

```diff
@@ -44,6 +44,14 @@ export class HeadlessCommerceDeliveryCartApiHelper {
 		);
 	}
 
+	async getCartItems(cartId: number) {
+		const getCartItems = await this.apiHelpers.get(
+			`${this.apiHelpers.baseUrl}${this.basePath}/carts/${cartId}/items`
+		);
+
+		return getCartItems;
```

Comment:
is there any reason on why we cannot return directly the this.apiHelper.. like the getComments method below?